src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -117,12 +117,16 @@ public void shootAtPlayer() {
     public void ezShot(){//still needs code to stop from shooting same tile twice. Consider using code in else statement in hardShot()
         int max = 10;
         int min = 1;
-        Random random = new Random();
-        int randRow = random.nextInt(max - min + 1) + min;
-        int randCol = random.nextInt(max - min + 1) + min;
+        Coordinate coor = new Coordinate(0, 0);
+        do {
+            Random random = new Random();
+            int randRow = random.nextInt(max - min + 1) + min;
+            int randCol = random.nextInt(max - min + 1) + min;
+            coor = new Coordinate(randRow, randCol);
+        }while(!goodShot(coor));
 
-        Coordinate coor = new Coordinate(randRow,randCol);
         playerShot(coor);
+        return;
```

Comment:
Totally not a big deal either way, but Java should call 'return' for you when it reaches the end of a void function